spec/integ/matrix-client-syncing.spec.ts

```diff
@@ -274,6 +274,15 @@ describe("MatrixClient syncing", () => {
 
             expect(fires).toBe(1);
         });
+
+        it("should work when all network calls fail", async () => {
+            httpBackend!.when("GET", "").fail(0, new Error("CORS or something"));
```

Comment:
```suggestion
			httpBackend!.expectedRequests = [];
            httpBackend!.when("GET", "").fail(0, new Error("CORS or something"));
```

The beforeEach sets up a bunch of successful responses which need overriding